esteid/urls.py

```diff
@@ -19,16 +21,19 @@
 
 urlpatterns = []
 
-if settings.DEBUG:
+if settings.ROOT_URLCONF == __name__:
```

Comment:
this looks odd.

Reply:
I thought this would help avoid accidentally including these urls in a project. But on a second thought, it might be actually useful to, so reverted.